src/controller/CHIPDeviceController.cpp

```diff
@@ -1383,11 +1390,11 @@ CHIP_ERROR DeviceCommissioner::CloseBleConnection()
 
 void DeviceCommissioner::OnSessionEstablishmentTimeout()
 {
+    // This is called from teh session establishment timer. Please see
```

Comment:
Please see which?  Also "the session" (typo).

Reply:
see https://github.com/project-chip/connectedhomeip/issues/14650 - this timer is a weird leftover bit that should be removed